deltachat-ios/Controller/AccountSetup/Instand Onboarding/InstantOnboardingViewController.swift

```diff
@@ -1,78 +1,50 @@
 import UIKit
 import DcCore
 
-class InstantOnboardingViewController: UITableViewController, MediaPickerDelegate, ProgressAlertHandler {
 
-    private struct SectionConfigs {
-        let headerTitle: String?
-        let footerTitle: String?
-        let cells: [UITableViewCell]
-    }
+class InstantOnboardingViewController: UIViewController, ProgressAlertHandler {
 
     private let dcContext: DcContext
     private let dcAccounts: DcAccounts
     weak var progressAlert: UIAlertController?
     var progressObserver: NSObjectProtocol?
 
-    private lazy var mediaPicker: MediaPicker? = {
+    var contentView: InstantOnboardingView { view as! InstantOnboardingView }
```

Comment:
there is a linter warning about the force cast - on github as well as in xcode.

in general, we're not using force cast because of bad experiences in the past - surely, it is useful here and there, but these things tend to sneak in to places where they really should not be used and cause real issues then.

not sure about the best graceful way, but a ...

```suggestion
    var contentView: InstantOnboardingView? { view as? InstantOnboardingView }
```

... and replacing the few usages by `contentView?` seems okay. alternatively, we could do a `guard` and crash explicitly if that fails.